submissions/yaripey/html-css-popup/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <title>Popup task</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <a href="#" class="nav-item">Mail</a>
+      <a href="#" class="nav-item">Images</a>
+      <div class="nav-item menu">
+        <label class="menu-button">
+          <input type="checkbox" class="menu-check">
+          <span class="menu-icon"></span>
+          <span class="popupmenu-wrap">
+            <section class="popupmenu">
```

Comment:
would be better to add markup `nav > ul > li` for dropdown menu

Reply:
You mean instead of `section` should be `nav`?